pkg/apis/apps/v1alpha1/servicebinding_types.go

```diff
@@ -8,8 +8,8 @@ import (
 
 // Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 
-// ServiceBindingRequestSpec defines the desired state of ServiceBindingRequest
-type ServiceBindingRequestSpec struct {
+// ServiceBindingSpec defines the desired state of ServiceBindingSpec
+type ServiceBindingSpec struct {
```

Comment:
The filename is still `servicebindingrequest_types.go`. It should be changed to `servicebinding_types.go`